tests/unit_tests/udf_wrapper_params/sequence_classification/mock_sequence_tokenizer.py

```diff
@@ -1,8 +1,11 @@
+from typing import List
+
 
 class MockSequenceTokenizer:
-    def __new__(cls, text: str, return_tensors: str):
-        return {'texts': text}
+    def __new__(cls, first_texts: List[str], second_texts: List[str] = None,
+                return_tensors: str = None):
+        return {'first_texts': first_texts}
```

Comment:
Why do we only return first_texts

Reply:
We can also return the second text, but only the first one will suffice because they both have the same length.

Reply:
ah, you are right, I know remember, how the mock worked